.github/workflows/sync_docs.yaml

```diff
@@ -1,33 +1,18 @@
-name: Sync docs from Discourse
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+name: Sync Discourse docs
 
 on:
   workflow_dispatch:
   schedule:
-    - cron: '53 0 * * *' # Daily at 00:53 UTC
-  push:
```

Comment:
Any special reason to not run on push to main?

Reply:
It could be done, but I think it's unnecessary. The workflow creates (or updates) a separate PR that would get merged anywhere between that day and the next few days. So a higher frequency than that might be a bit of overkill.